tbx/static_src/sass/components/_showcase.scss

```diff
@@ -120,7 +120,9 @@
             gap: $spacer-mini;
         }
 
+        // home page showcase block only
         &--with-tail {
+            @include font-size(heading-three-b);
```

Comment:
Nitpick - this isn't really linked to the `--with-tail` classname as the size is irrelevant to the arrow after. We could add another alternative variant here - but really it's not a big deal so fine to leave as is too.